android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -366,9 +366,8 @@ public void loadJavascript(String func) {
 
   public void callJavascriptAfterLoad() {
     if(this.javascriptAfterLoad.size() > 0) {
-      // Don't call this WebView method on just ANY thread - run it on the main UI thread.
-      // https://stackoverflow.com/a/22611010
-      this.postDelayed(new Runnable() {
+      Handler handler = new Handler();
```

Comment:
![On this constructor's deprecation(https://github.com/user-attachments/assets/ecd9e63a-7987-4c96-a9b0-d0097e2ac598)

(from https://developer.android.com/reference/android/os/Handler)

The docs strongly suggest making this, at least, `new Handler(Looper.getMainLooper())`.

Reply:
After a bit more searching, I found this:  https://stackoverflow.com/questions/13974661/runonuithread-vs-looper-getmainlooper-post-in-android

The big takeaway from that link:  `Looper.getMainLooper()` will return the Looper instance associated with the primary UI thread, which is what we want.